app/pages/api/quoteEmail.ts

```diff
@@ -0,0 +1,37 @@
+// LIBRARY IMPORTS
+import type { NextApiRequest, NextApiResponse } from "next";
+import nodemailer from "nodemailer";
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
This doesn't appear to be named. It should probably be like this:
`export default async function QuoteEmails(req: NextApiRequest, res: NextApiResponse) {`...